resources/js/package.js

```diff
@@ -37,6 +37,6 @@ async function updateAddressFromExperiusPostcode(address) {
         return
     }
 
-    set(address, 'city', response.data.postcode.city)
-    set(address.street, 0, response.data.postcode.street)
+    window.app.checkout[type + '_address'].city = response.data.postcode.city
```

Comment:
Unfortunately this will break functionality in the customer address forms https://github.com/rapidez/account/blob/b290d244fe58149e00a99deffcfc87d5aecf7621/resources/views/account/partials/address-form.blade.php#L60